variables.tf

```diff
@@ -52,6 +52,28 @@ variable "appautoscaling_settings" {
   type        = map(any)
 }
 
+variable "capacity_provider_strategy" {
+  default     = null
+  description = "List of capacity provider strategies."
+  type = list(object({
+    capacity_provider = string
+    weight            = optional(string, null)
+    base              = optional(string, null)
+  }))
+}
+
+variable "deployment_circuit_breaker" {
+  default = {
+    enable   = false
+    rollback = false
+  }
+  description = "Deployment circuit breaker configuration."
+  type = object({
+    enable   = bool
+    rollback = optional(bool, false)
```

Comment:
this is required according to https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecs_service#deployment_circuit_breaker, IMO we should keep it that way